### ISSUE TITLE
Implement a tag listing layout and refine tagging support for the design guide

### ISSUE THREAD

**Issue by klepas** [AusDTO/gov-au-ui-kit#423]
Tagging is implemented (eg. adding a tag to a component in the SASS and having the right tags parsed into Jekyll).

What’s still needed:
- create an index list view/layout for all items tagged with a given tag
- possibly refine by being able to select more than one tag at a time (a nice to have atm).

**Comment by klepas** [comment]
This can wait until aftre #427 is completed, IMHO.

**Comment by klepas** [comment]
Blocked by the following additional tasks (at least the first one)
- [ ] decide on a taxonomy of tags
- [x] add tags throughout our ui-kit codebase to all the features

**Comment by klepas** [comment]
Noting that we now have the styling available for this — when latest `develop` is merged over 1.10.3  we will have tagging available via `.tags`.

This should also be a ticket in the Design Guide repo instead.

**Comment by klepas** [comment]
The remaining work in this ticket applies to the design guide, which has already got a tix capturing the work there: https://github.com/AusDTO/dto-design-guide/issues/63

Closing this subsequently.
